pkg/sentry/fsimpl/devpts/line_discipline.go

```diff
@@ -393,6 +401,10 @@ func (*inputQueueTransformer) transform(l *lineDiscipline, q *queue, buf []byte)
 			if l.termios.IEnabled(linux.INLCR) {
 				cBytes[0] = '\r'
 			}
+		case l.termios.ControlCharacters[linux.VINTR]: // ctrl-c
+			l.fgProcessGroup.SendSignal(kernel.SignalInfoPriv(linux.SIGINT))
+		case l.termios.ControlCharacters[linux.VSUSP]: // ctrl-z
+			l.fgProcessGroup.SendSignal(kernel.SignalInfoPriv(linux.SIGTSTP))
```

Comment:
Linux also handles SIGQUIT. Any reason not to add that here?

https://elixir.bootlin.com/linux/v5.6/source/drivers/tty/n_tty.c#L1291

Reply:
no, I can add it